packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/main_dev.dart

```diff
@@ -15,20 +15,28 @@ void main() async {
   const environment = EnvironmentConfig.development();
 
   await configureApp(environment);
-
-  runApp({{^enable_dev_menu}}const{{/enable_dev_menu}} {{project_name.pascalCase()}}(
-    config: environment,{{#enable_dev_menu}}
-    createDebugMenuInstance: (context, widget, navKey) =>
-        AppDevMenuGestureDetector.withDependencies(
-      context,
-      navKey,
-      child: widget,
-      onDevMenuPresented: () {
-        showAppDevMenuBottomSheet(
-          navKey.currentContext!,
-        );
-      },
-    ),
+{{#enable_dev_menu}}
```

Comment:
This `if-else` approach is quite ugly. We do not need two `runApp` calls.

You can extract this `createDebugMenuInstance` function to a local variable and then pass it as a value when the menu needs to be enabled. Take this pseudocode for example:

```dart

final _debugMenuBuilder = (context, widget, navKey) =>
    AppDevMenuGestureDetector.withDependencies(...);


runApp(
  config: environment,
  createDebugMenuInstance: EnvironmentConfig.enableDevMenu ? _debugMenuBuilder : null;
);
```

I also have a problem with the name of the `createDebugMenuInstance` parameter.
In some places it is called dev menu and on other places it is debug menu?
Also this parameter is essentialy a builder (a common flutter pattern) so the name should reflect that.

All this also applies to the main staging file.